src/pages/components/button/usage.mdx

```diff
@@ -542,107 +539,805 @@ recommended action labels.
+secondary buttons.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do use the secondary button in conjunction with the primary button.">
+
+![Do use the secondary button in conjunction with the primary button.](images/button_usage_secondary_2.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not use the secondary button in isolation or for a positive/primary action.">
+
+![Do not use the secondary button in isolation or for a positive/primary action.](images/button_usage_secondary_3.png)
+
+  </DoDont>
+</DoDontRow>
+
+## Tertiary button
+
+Tertiary buttons have less prominence than a primary button and slightly more
+prominence than a ghost button. Tertiary buttons work well on their own or as
+part of a button group.
+
+<Row>
+<Column colLg={12}>
+
+![Tertiary button in the context of a product UI.](images/button_usage_tertiary_1.png)
+
+</Column>
+</Row>
+
+<Caption>Tertiary button in the context of a product UI.</Caption>
+
+Tertiary buttons should stay aligned with the content of the page. Always make
+sure the tertiary button has adequate padding on all sides. As mentioned in the
+[fluid button guidance](#alignment), tertiary buttons should not be used in
+fluid arrangements.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do align the tertiary button with the rest of the content.">
+
+![Do align the tertiary button with the rest of the content.](images/button_usage_tertiary_2.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not use a tertiary button in a fluid arrangement.">
+
+![Do not use a tertiary button in a fluid arrangement.](images/button_usage_tertiary_3.png)
+
+  </DoDont>
+</DoDontRow>
+
+### Best practices
+
+#### Tertiary buttons for page headers
+
+It is challenging to use a primary button in the header of a page because the
+content beneath the header is probably going to have a primary action, or will
+in a future release. Even if the the button in the page header is not styled as
+a primary button, it has significant prominence due to its hierarchical
+placement at the top.. Therefore, it is advised to use a tertiary button for
+page headers.
+
+If it is determined that the button in the page header, across all tabs, should
+be primary, ensure none of the content below the header contain another primary
+action.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do use a tertiary button in the page header if there is a primary button in the content section.">
+
+![Do use a tertiary button in the page header if there is a primary button in the content section.](images/button_usage_tertiary_4.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not use a primary button in the page header and the content section.">
+
+![Do not use a primary button in the page header and the content section.](images/button_usage_tertiary_5.png)
+
+  </DoDont>
+</DoDontRow>
+
+#### Tertiary buttons in button groups
+
+In [Button groups](#button-groups), where there is one primary and two other
```

Comment:
I think since this isn't directly reference the "Button group" heading it can just be lower case. 

```suggestion
In [button groups](#button-groups), where there is one primary and two other
```